.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/javascript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="16-buster"
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
+
+# Install basic packages useful for development 
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends rsync git m4 build-essential patch unzip wget pkg-config libgmp-dev libev-dev libhidapi-dev libffi-dev opam jq zlib1g-dev inotify-tools tmuxinator
+
+# [Optional] Uncomment if you want to install an additional version of node using nvm
+# ARG EXTRA_NODE_VERSION=10
+# RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
+
+# Install esy
+RUN su node -c "sudo npm install -g --unsafe-perm esy"
+
+# Install LIGO
+RUN wget https://ligolang.org/deb/ligo.deb && sudo apt install ./ligo.deb
```

Comment:
Does LIGO provide exact compiler version and a checksum of the .deb file? This could be made more secure.

Reply:
At the very least, it will serve as an integrity check (say, if the there was a network failure during the download?)

Reply:
Also for ligo there is no need to download a `.deb` we could just download the statically linked binary

Reply:
> Does LIGO provide exact compiler version and a checksum of the .deb file? This could be made more secure.

I don't think they do, but I could generate a checksum myself and check it against that (would need to be changed every time there's a new version of ligo though)

> Also for ligo there is no need to download a .deb we could just download the statically linked binary

Is that better?

Reply:
It is, btw you can also put the binary on the repository, that's okay